### PR TITLE
Update getting started

### PR DESCRIPTION
**Pull request by carlbuchmann**
Update getting started Readme, clarifying some steps.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-![Arista CVP Automation](https://img.shields.io/badge/Arista-CVP%20Automation-blue) ![Arista EOS Automation](https://img.shields.io/badge/Arista-EOS%20Automation-blue)
+![AristaCloudVision Automation](https://img.shields.io/badge/Arista-CVP%20Automation-blue) ![Arista EOS Automation](https://img.shields.io/badge/Arista-EOS%20Automation-blue)
```
Comment (titom73) [inline README.md]: Space missing between Arista and CloudVision

File: README.md [hunk 2]
```diff
@@ -39,38 +39,63 @@ The ATD Lab topology consists of 2 Spines, 4 Leafs and 2 Hosts, as shown below.
...
+
+    ```shell
+    # Run audit playbook to validate Fabric states
+    $ ansible-playbook playbooks/atd-validate-states.yml
+
+    # Execute EOS_SNAPSHOT role to collect show commands
+    $ ansible-playbook playbooks/atd-snapshot.yml
+    ```
+
+    - Review generated output.
```
Comment (titom73) [inline README.md:102]: One blank line is missing

**Review by titom73 — DISMISSED** [review]

File: README.md [hunk 3]
```diff
@@ -39,38 +39,63 @@ The ATD Lab topology consists of 2 Spines, 4 Leafs and 2 Hosts, as shown below.
 
 Connect to your ATD Lab environment.  If you need an ATD Lab instance, please contact your local account team.  Once connected to the ATD Lab instance, select the Programmability IDE.  This container is built with all the necessary requirements and python modules to run AVD playbooks.
```
Comment (onurgashi) [inline README.md]: . If you need
. Once
. This container

**Review by onurgashi — APPROVED** [review]
LGTM!

File: README.md (hunk 3 above)
Comment (ankudinov) [inline README.md]: python -> Python

File: README.md [hunk 4]
```diff
@@ -39,38 +39,63 @@ The ATD Lab topology consists of 2 Spines, 4 Leafs and 2 Hosts, as shown below.
...
-```shell
-# Setup your git global config (optional)
-git config --global user.email "you@example.com"
-git config --global user.name "Your Name"
+    - Open terminal window in vscode View -> Terminal from menu, and run the following commands:
 
-# Run Script to setup environment
-curl -fsSL https://get.avd.sh/atd/install.sh | sh
+    ```shell
+    # Setup your git global config (optional)
```
Comment (danieltudares) [inline README.md:50]: Consider removing this step. Although it's marked as optional, it's really not necessary and unrelated.

File: README.md [hunk 5]
```diff
@@ -39,38 +39,63 @@ The ATD Lab topology consists of 2 Spines, 4 Leafs and 2 Hosts, as shown below.
 
 Connect to your ATD Lab environment.  If you need an ATD Lab instance, please contact your local account team.  Once connected to the ATD Lab instance, select the Programmability IDE.  This container is built with all the necessary requirements and python modules to run AVD playbooks.
 
+1. Setup Git user and email and setup the ATD lab environement.
```
Comment (ankudinov) [inline README.md]: remove 2nd setup: "Setup Git user and email and the ATD lab environement."

File: README.md [hunk 6]
```diff
@@ -39,38 +39,63 @@ The ATD Lab topology consists of 2 Spines, 4 Leafs and 2 Hosts, as shown below.
...
 
-```shell
-# Setup your git global config (optional)
-git config --global user.email "you@example.com"
-git config --global user.name "Your Name"
+    - Open terminal window in vscode View -> Terminal from menu, and run the following commands:
```
Comment (ankudinov) [inline README.md]: vscode -> VS Code

File: README.md [hunk 7]
```diff
@@ -39,38 +39,63 @@ The ATD Lab topology consists of 2 Spines, 4 Leafs and 2 Hosts, as shown below.
...
+    $ curl -fsSL https://get.avd.sh/atd/install.sh | sh
+    ```
 
-# Update Inventory with Lab Credentials
-edit credentials in vscode: atd-avd/atd-inventory/inventory.yml
+2. Update Inventory with Lab Credentials
 
-# Run Playbook to Prepare CloudVision for AVD
-$ ansible-playbook playbooks/atd-prepare-lab.yml
+    - Open file in vscode: atd-avd/atd-inventory/inventory.yml
```
Comment (ankudinov) [inline README.md]: VS Code

File: README.md [hunk 8]
```diff
@@ -39,38 +39,63 @@ The ATD Lab topology consists of 2 Spines, 4 Leafs and 2 Hosts, as shown below.
...
 
-# Execute Tasks in CVP manually
+    - Edit credentials with the ones provide on test drive landing page.
```
Comment (ankudinov) [inline README.md]: - Set credentials in the `inventory.yml` to the credentials on the Arista Test Drive landing page.

File: README.md [hunk 9]
```diff
@@ -39,38 +39,63 @@ The ATD Lab topology consists of 2 Spines, 4 Leafs and 2 Hosts, as shown below.
...
 
-# Run Playbook to Deploy AVD Setup
-$ ansible-playbook playbooks/atd-fabric-deploy.yml
 
-# Execute Tasks in CVP manually
+3. Run Playbook to Prepare CloudVision for AVD
```
Comment (ankudinov) [inline README.md]: the

File: README.md [hunk 10]
```diff
@@ -39,38 +39,63 @@ The ATD Lab topology consists of 2 Spines, 4 Leafs and 2 Hosts, as shown below.
...
 
-# Run audit playbook to validate Fabric states
-$ ansible-playbook playbooks/atd-validate-states.yml
+    - Execute the following commands:
 
-# Execute EOS_SNAPSHOT role to collect show commands
-$ ansible-playbook playbooks/atd-snapshot.yml
-```
+    ```shell
+    # Move to directory
```
Comment (ankudinov) [inline README.md]: `# change to ATD-AVD dicrectory`

Comment (ankudinov) [reply README.md]: directory =)

File: README.md [hunk 11]
```diff
@@ -39,38 +39,63 @@ The ATD Lab topology consists of 2 Spines, 4 Leafs and 2 Hosts, as shown below.
...
+    $ cd labfiles/arista-ansible/atd-avd
 
+    # Run Playbook to Prepare CloudVision for AVD
+    $ ansible-playbook playbooks/atd-prepare-lab.yml
+    ```
+
+ - Verify that tasks in CloudVision have executed automatically.
```
Comment (ankudinov) [inline README.md]: Need a native speaker here, but I think it's `have been executed`?

File: README.md [hunk 12]
```diff
@@ -39,38 +39,63 @@ The ATD Lab topology consists of 2 Spines, 4 Leafs and 2 Hosts, as shown below.
...
+
+4. Run playbook to deploy AVD setup
+
+    - Execute the following commands:
+
+    ```shell
+    # Run Playbook to Deploy AVD Setup
+    $ ansible-playbook playbooks/atd-fabric-deploy.yml
+    ```
+    -  Execute Tasks in CloudVision manually.
```
Comment (ankudinov) [inline README.md]: Execute pending tasks on the CloudVision Portal manually.

File: README.md [hunk 13]
```diff
@@ -39,38 +39,63 @@ The ATD Lab topology consists of 2 Spines, 4 Leafs and 2 Hosts, as shown below.
...
+ - Verify that tasks in CloudVision have executed automatically.
+
+4. Run playbook to deploy AVD setup
+
+    - Execute the following commands:
+
+    ```shell
+    # Run Playbook to Deploy AVD Setup
+    $ ansible-playbook playbooks/atd-fabric-deploy.yml
+    ```
```
Comment (tgodaA) [inline README.md:86]: Blank line missing after quotes, [md032](https://github.com/DavidAnson/markdownlint/blob/v0.24.0/doc/Rules.md#md032)

File: README.md [hunk 14]
```diff
@@ -39,38 +39,63 @@ The ATD Lab topology consists of 2 Spines, 4 Leafs and 2 Hosts, as shown below.
...
 
-# Run Playbook to Prepare CloudVision for AVD
-$ ansible-playbook playbooks/atd-prepare-lab.yml
+    - Open file in vscode: atd-avd/atd-inventory/inventory.yml
 
-# Execute Tasks in CVP manually
+    - Edit credentials with the ones provide on test drive landing page.
 
-# Run Playbook to Deploy AVD Setup
-$ ansible-playbook playbooks/atd-fabric-deploy.yml
```
Comment (tgodaA) [inline README.md:64]: Extra blank line 61. Line 60 is also blank.

**Comment by mtache** [comment]
Could we add a line about changing [this knob](https://github.com/arista-netdevops-community/atd-avd/blob/master/atd-inventory/group_vars/ATD_LAB.yml#L16) when using vEOS ? I get caught in that trap every single time :)

**Comment by mtache** [comment]
Could we add a line about changing [this knob](https://github.com/arista-netdevops-community/atd-avd/blob/master/atd-inventory/group_vars/ATD_LAB.yml#L16) when using vEOS ? I get caught in that trap every single time :)

File: README.md (hunk 9 above)
Comment (carlbuchmann) [reply README.md]: corrected

**Comment by carlbuchmann** [comment]
> Could we add a line about changing [this knob](https://github.com/arista-netdevops-community/atd-avd/blob/master/atd-inventory/group_vars/ATD_LAB.yml#L16) when using vEOS ? I get caught in that trap every single time :)

Comment on line 36:
```
Current repository is built with cEOS management interface (`Management0`). If you run a vEOS topology, please update `mgmt_interface` field to `Management1` in [inventory](./atd-inventory/group_vars/ATD_LAB.yml)
```

**Pull request merged**